scripts/config.js

```diff
@@ -5,41 +5,42 @@ const DEMO_DIRECTORY = path.join("demo");
 const DISTRIBUTION_DIRECTORY = path.join("dist");
 
 export default {
-  "versions": {
-    "rn": "0.64.0",
-    "node": "14.15.4"
+  versions: {
+    rn: "0.64.0",
+    node: "14.15.4",
   },
-  "constants": {
+  constants: {},
+  scaffold: {
+    directory: SCAFFOLD_DIRECTORY,
   },
-  "scaffold": {
-    "directory": SCAFFOLD_DIRECTORY,
-  },
-  "dist": {
-    "directory": DISTRIBUTION_DIRECTORY,
-    "builds": {
-      "raw": {
-        "directory": path.join(DISTRIBUTION_DIRECTORY, "raw"),
+  dist: {
+    directory: DISTRIBUTION_DIRECTORY,
+    builds: {
+      raw: {
+        directory: path.join(DISTRIBUTION_DIRECTORY, "raw"),
         // Avoids react-native cli error:
         // "Project name shouldn't contain "HelloWorld" name in it,
```

Comment:
```suggestion
        // "Project name shouldn't contain "ProjectName" name in it,
```

Reply:
Understood! That makes sense. I did try to review the js files but wasn't 100% sure if that was a requirement. Thanks for pointing it out.

Reply:
What about the change in `scripts/config.js`? What is that for?

Reply:
Its just an old comment that should have been updated and was suggesting you to include it in your PR since its related. This new github suggestions allows you to commit really easy by just pressing the button :) it works nice for small changes